### DISCUSSION TITLE
Trigger actions on error

### DISCUSSION THREAD

**Discussion by joaogabrieldasilva** [react-hook-form/react-hook-form#8363] [Q&A]
Hey guys, I'm kinda new to RHF, but basically I'd like to know if there is a way to trigger functions on Error.

I have a select input on React Native and as it's not a real input (just a modal that opens when user clicks on the input field). So I was wondering if there is a way to open this input onError like TextInput onFocus does on error

**Comment by bluebill1049** [comment]
could you sharea a simple codesandbox?

**Comment by Moshyfawn** [comment]
The first thing that comes to mind is to call your modal open inside useEffect on `errors` change and checking your select error
